example/main.js

```diff
@@ -45,6 +45,7 @@ const MAPBOX_ACCESS_TOKEN = process.env.MAPBOX_ACCESS_TOKEN ||
   'Set MAPBOX_ACCESS_TOKEN environment variable or put your token here.';
 
 const CHOROPLETHS_FILE = './example/data/sf.zip.geo.json';
+const EXTRUDED_CHOROPLETHS_FILE = './example/data/sfo.buildings.zoom13.json';
```

Comment:
This file is just several hundred kilo... similar to sf.zip.geo.json file. I guess we could get away with it for now. Or, I could generate a smaller file.